eng/scripts/cleanup-allowlist.txt

```diff
@@ -10,3 +10,38 @@ AzSecPackAutoConfigRG
 
 static-test-resources
 LiveTestSecrets
+
+# Groups that may or may not be needed as static test resources. Eventually this list should be pruned down to zero, with all resources living in `static-test-resources`
```

Comment:
Should we instead add a tag to these resources instead of listing them here? Or is the goal that you want to use this as a baseline and a work list to clean-up overtime but to stop any new ones creeping in?

Reply:
Yeah I think that would be a fine approach. I figured it was slightly easier to track by recording them here, i.e. I'm worried we'll forget about them if we tag them (or if they get re-provisioned). But it is more consistent with our long term strategy.